src/main/java/duke/TaskItem.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+
+public class TaskItem {
+
+    private String task;
+    private boolean completed;
+
+    /**
+     * Constructor for creating a duke.TaskItem object.
+     * @param task the description of the duke.TaskItem.
+     */
+    public TaskItem(String task) {
+        this.task = task;
+        this.completed = false; // by default if you add it to the list then it should be false first.
+    }
+
+    /**
+     * describes the task.
+     * @return returns the description of the duke.TaskItem as a String.
+     */
+    public String describeTaskItem() {
+        return this.task;
+    }
+
+    /**
+     * queries whether the task is completed or not.
+     * @return a boolean to tell whether or not the task is completed.
+     */
+    public boolean isCompleted() {
+        return this.completed;
+    }
+
+    /**
+     * Completes this task.
+     */
+    public void completeTask() {
+        this.completed = true;
+    }
+
+    /**
+     * toFileString returns the TaskItem in a string.
+     * @return returns the Deadline in a string format that is specifically made for the dukeData.txt file.
+     */
+    public String toFileString() {
```

Comment:
This method seems redundant since you already have a `toString()` method that contains the same lines of code.